tests/unit/plugins/module_utils/test_system_access_users_utils.py

```diff
@@ -698,31 +852,26 @@ def test_generate_hashed_secret_failure_invalid_hash(mock_run_function):
 )
 def test_generate_hashed_secret_error_in_crypt(mock_run_function):
     mock_run_function.return_value = {"stdout": "", "stderr": "error in crypt function"}
-
     user = User(name="test", password="test")
     with pytest.raises(OPNSenseCryptReturnError) as excinfo:
```

Comment:
```suggestion
    with pytest.raises(OPNsenseCryptReturnError) as excinfo:
```

Reply:
Thanks for the suggestion, fixed in cfcd2b4439517a3f4a6370ccb6f56fa95486db52.